index.html

```diff
@@ -131,197 +131,18 @@ <h3 class="connecting">LET'S CONNECT</h3>
-                                    </li>
-                                </ul>
-                                <p class="paragraph">
-                                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.
-                                </p>
-                                <ul class="language-button">
-                                    <li><button type="button" class="l-button">html</button></li>
-                                    <li><button type="button" class="l-button">css</button></li>
-                                    <li><button type="button" class="l-button">javaScript</button></li>
-                                </ul>
-                                <button type="button" class="see-project">See Project</button>
-                            </div>                    
-                        </div>
-                        <!-- Desktop only version of works card -->
-                        <div class="card card-desk">
-                            <div class="col2">
-                                <h2 class="main-heading">
-                                    Multi-Post Stories
-                                </h2>
-                                <ul class="top-container">
-                                    <li>
-                                        <p class="canopy">CANOPY</p>
-                                    </li>
-                                    <li><img src="./pics/dot.png" alt="dot"></li>
-                                    <li>
-                                        <p class="dev">Front End Dev</p>
-                                    </li>
-                                    <li><img src="./pics/dot.png" alt="dot"></li>
-                                    <li>
-                                        <p class="dev year">2021</p>
-                                    </li>
-                                </ul>
-                                <p class="paragraph">
-                                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.
-                                </p>
-                                <ul class="language-button">
-                                    <li><button type="button" class="l-button">html</button></li>
-                                    <li><button type="button" class="l-button">css</button></li>
-                                    <li><button type="button" class="l-button">javaScript</button></li>
-                                </ul>
-                                <button type="button" class="see-project">See Project</button>
-                            </div>
-                            <img class="screenshot" src="./pics/homepage4.png" alt="work4">
-                        </div>               
-                    </div>    
+                <!--Popup-->
+                <section class="detail-section">
```

Comment:
- [ ] In mobile, the design does not follow the Figma closely:

![image](https://user-images.githubusercontent.com/57421823/156442947-e97aee30-3deb-4e72-8060-7ffc5bfbb778.png)

![image](https://user-images.githubusercontent.com/57421823/156443027-ebe99185-6b72-41d3-bcd9-4d0157497cbe.png)

Figma:

![image](https://user-images.githubusercontent.com/57421823/156443084-14b6cb7f-f83e-4f92-a0d4-c308f196b5dc.png)

Please, closely follow the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) given design implementing margins, padding, font styles, and Flexbox where is proper.

- [ ] The technologies row (HTML, CSS, javascript, etc) repeats itself.  Please, kindly correct this so the popup info is consistent 💪

Reply:
Thanks it has been resolved